### PR TITLE
adds a carp shank

### PR DESCRIPTION
**Pull request by boodaliboo**
adds a new shank that is craftable with carp tooth 5 cable coil a torn cloth and a metal rod
in order to obtain a carp tooth you must butcher a space carp
this shank does 11 damage and injects 1 unit of carpotoxin per hit
#### Changelog
image

:cl:  
rscadd: Added carp tooth and carp shank
tweak: tweaked space carp drops added carp tooth  
imageadd: added two new sprites
/:cl:

### CODE REVIEW

**Comment by ynot01** [comment]
thats a lot of damage but then again you can print a butcher's cleaver for 15 force idk bro

if you sharpen it itll deal 21 damage

**Comment by BOBsHERE** [comment]
it could also apply a small amount of carpotoxin on hit

**Comment by boodaliboo** [comment]
what you reccomend its damage be and eh not sure on carpotoxin cant remeber how deadly it is

**Comment by Marmio64** [comment]
Make the shank deal 15 damage instead so its on par with a survival knife and bone dagger

**Comment by BOBsHERE** [comment]
carpotoxin is a moderate poison on its own, its 2 damage per tick. If you lower the shank brute damage but make it apply some carpotoxin then its not just another knife and its unique and worth crafting.

**Comment by boodaliboo** [comment]
yeah let me see

**Comment by boodaliboo** [comment]
> carpotoxin is a moderate poison on its own, its 2 damage per tick. If you lower the shank brute damage but make it apply some carpotoxin then its not just another knife and its unique and worth crafting.



> Make the shank deal 15 damage instead so its on par with a survival knife and bone dagger

done

**Comment by Marmio64** [comment]
if you're having it deal carpotoxin should probably make it only deal 10-12 damage or something.

**Comment by boodaliboo** [comment]
> if you're having it deal carpotoxin should probably make it only deal 10-12 damage or something.

done

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -1,717 +1,744 @@
...
+	icon_state = "shank"
+	force = 11
+	throwforce = 14
+	throw_speed = 1
+	throw_range = 1
+	attack_verb = list("poked", "stabbed")
+	
+/obj/item/shank/afterattack(atom/target, mob/user, proximity = TRUE)
+	. = ..()
+	user.changeNext_move(CLICK_CD_RAPID)
```
Comment (SomeguyManperson) [inline code/game/objects/items/weaponry.dm]: 🤔

**Review by SomeguyManperson — CHANGES_REQUESTED** [review]
also line ending issues by the looks of it

File: code/datums/components/crafting/recipes.dm [hunk 2]
```diff
@@ -745,6 +745,13 @@
 					/obj/item/reagent_containers/food/snacks/grown/pumpkin = 1,
 					/obj/item/stack/cable_coil = 5)
 	category = CAT_MISC
+	
+/datum/crafting_recipe/aispook
```
Comment (Hopekz) [inline code/datums/components/crafting/recipes.dm:749]: What is aispook? Why is this the path?
AI spook?
ais pook?
A is pook?

File: code/game/objects/items/weaponry.dm [hunk 3]
```diff
@@ -1,717 +1,744 @@
...
+/obj/item/extendohand/attack(atom/M, mob/living/carbon/human/user)
+	var/dist = get_dist(M, user)
+	if(dist < min_reach)
+		to_chat(user, "<span class='warning'>[M] is too close to use [src] on.</span>")
+		return
+	M.attack_hand(user)
+
+/obj/item/tooth
+	name = "carp tooth"
+	desc = "a tooth harvested from a space carp."
```
Comment (Ktlwjec1) [inline code/game/objects/items/weaponry.dm]: A

File: code/game/objects/items/weaponry.dm [hunk 4]
```diff
@@ -1,717 +1,744 @@
...
+	icon_state = "shark_tooth"
+	force = 2
+	throwforce = 1
+	throw_speed = 1
+	throw_range = 1
+	attack_verb = list("poked", "stabbed")
+
+/obj/item/shank
+	name = "carp shank"
+	desc = "a shank made out of a metal rod, cable coil, and a carp tooth."
```
Comment (Ktlwjec1) [inline code/game/objects/items/weaponry.dm]: A

**Review by alexkar598 — CHANGES_REQUESTED** [review]
get. your. line. endings. together.

**Comment by boodaliboo** [comment]
> get. your. line. endings. together.

i have no clue what you mean by this

**Pull request closed without merging**